src/region/RegionAnnotations.tsx

```diff
@@ -9,10 +9,12 @@ import { CreatorItem, CreatorStatus } from '../store/creator';
 import { scaleShape } from './regionUtil';
 import './RegionAnnotations.scss';
 
+type Region = { location: number; message: string; shape: Rect };
```

Comment:
There's a matching type alias actually available in `region/actions` and `creator/types`. Can we consolidate to CreatorItem?